galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -2352,6 +2418,36 @@ The excerpt below is an example of how this Galera parameter can be triggered fr
    SET GLOBAL wsrep_provider_options = 'socket.ssl_reload=1';
 
 
+.. _`sync_binlog`:
+.. rst-class:: section-heading
+.. rubric:: ``sync_binlog``
+
+.. index::
+   pair: wsrep Provider Options; sync_binlog
+
+Synchronously flush binary log to disk after every #th event. The options are:

```

Comment:
what is #th event ?

Reply:
It means, for example, every tenth event.